lib/OpenQA/Script/CloneJobSUSE.pm

```diff
@@ -5,10 +5,12 @@ package OpenQA::Script::CloneJobSUSE;
 use Mojo::Base -strict, -signatures;
 use Data::Dump 'pp';
 use Exporter 'import';
+use OpenQA::JobSettings;
 
 our @EXPORT = qw(detect_maintenance_update);
 
 sub collect_incident_repos ($url_handler, $settings) {
+    OpenQA::JobSettings::expand_placeholders($settings);
```

Comment:
This function should probably return whether some placeholders could not be expanded. Don't do this by checking for a  `%` as mentioned in the other comment.

Reply:
Ok. I suggest we do
* #6662

first to ensure that the worker has no problems with the return value of expanding and then I can try to incorporate that here.